src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/DueDiligenceValidator.java

```diff
@@ -114,6 +114,7 @@ private boolean validateDiligence(String diligence, Errors errors, String loggin
                 OverseasEntitySubmissionDto.DUE_DILIGENCE_FIELD,
                 DueDiligenceDto.DILIGENCE_FIELD);
         return StringValidators.isNotBlank(diligence, qualifiedFieldName, errors, loggingContext)
-                && StringValidators.isLessThanOrEqualToMaxLength(diligence, 256, qualifiedFieldName, errors, loggingContext);
+                && StringValidators.isLessThanOrEqualToMaxLength(diligence, 256, qualifiedFieldName, errors, loggingContext)
+                && StringValidators.isValidCharacters(diligence, qualifiedFieldName, errors, loggingContext);
```

Comment:
If I am correct the only value available for the following field is just the string `agree` as for spec [here](https://github.com/companieshouse/overseas-entities-api/blob/a78508edba73203e54357c96bccd4f3d2bc0a018/spec/schema.json#L589).

Reply:
Oh yeah I forgot the spec my bad, haven't worked on the api in a while. What does that agree mean?

Reply:
Oh I see, could there be extra validation required to check it matches the only valid string?

Reply:
It is just a check box [here](https://github.com/companieshouse/overseas-entities-web/blob/cfa31b37263edaa2b37bb163d5e54b2b9403cbe2/views/due-diligence.html#L75), and if the value is present should be just the `agree` string, even the length should not be necessary.

Reply:
I agree (sorry, couldn't resist!). So yes, sounds like we should just be checking the value is equal to 'agree', obviously handling null values too.

Reply:
So will this require a change in the bug or just create a new one entirely?

Reply:
For sure the fix should be as suggested, probably in the story need some changes on the expected behaviour including the fact that only the `agree` world should be accepted as for implementation and spec.